spec/api-auto-updater-spec.js

```diff
@@ -3,21 +3,15 @@ const { ipcRenderer } = require('electron')
 const { expect } = require('chai')
 
 describe('autoUpdater module', function () {
-  // XXX(alexeykuzmin): Calling `.skip()` in a 'before' hook
-  // doesn't affect nested 'describe's
   beforeEach(function () {
-    // Skip autoUpdater tests in MAS build.
-    if (process.mas) {
-      this.skip()
-    }
+    if (process.mas) this.skip()
   })
 
   describe('checkForUpdates', function () {
     it('emits an error on Windows when called the feed URL is not set', function (done) {
       if (process.platform !== 'win32') {
-        // FIXME(alexeykuzmin): Skip the test.
-        // this.skip()
-        return done()
+        this.test.parent.pending = true
```

Comment:
I'm not a fan of hitting this internal mocha API.

Might be slightly biased but I prefer the approach I started taking in #15782 with declarative `platformDescribe` and `platformIt` methods.